persistence-modules/hibernate-mapping-2/src/main/java/com/baeldung/hibernate/HibernateUtil.java

```diff
@@ -1,4 +1,6 @@
-package com.baeldung;
+package com.baeldung.hibernate;
```

Comment:
I moved the main classes under "hibernate" package to be consistent with the tests which are under it as well